README.md

```diff
@@ -622,15 +621,16 @@ The context contains:
 * `res` - An object containing details about the response. This is the [same interface as a response you'd receive from `fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Response).
 * `isCollection` - Whether the response is expected to be a collection.
 * `resources` - An array of resource URIs affected by the request.
-* `cacheStore` - A reference to the cache store
+* `cacheStore` - A reference to the cache store.
 
 If `res` is attached to the context before `next` is called, then a request will not be made. In order to attach a `res`, do the following:
 
 ```javascript
+const OK = 200;
```

Comment:
While I like constants in full blown code, in example snippets like this they just hide details.